src/main/kotlin/com/terraformation/backend/i18n/Messages.kt

```diff
@@ -147,6 +147,18 @@ class Messages {
 
   fun batchCsvQuantityInvalid() = getMessage("batchCsvQuantityInvalid")
 
+  fun monitoringPlotNortheastCorner(plotName: String) =
+      getMessage("monitoringPlotsNortheastCorner", plotName)
```

Comment:
The string key uses "plots" while the function name uses "plot" -- probably better to make them the same.